clusterbuster

```diff
@@ -1780,7 +1799,14 @@ function get_logs() {
 # YAML fragment generation
 ################################################################
 
-function default_security_context_content() {
+function privileged_security_context_content() {
+    cat <<'EOF'
+privileged: true
+runAsUser: 0
```

Comment:
Why we need to add it runAsUser: 0 ?
Is it part of privileges issue ?

Reply:
I think it may default to `runAsUser: 0` anyway if not set when using privileged.

Reply:
Yes, see above.